multicluster/controllers/multicluster/member_clusterset_controller.go

```diff
@@ -130,7 +114,7 @@ func (r *ClusterSetReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
-func (r *ClusterSetReconciler) validateLocalClusterClaim(clusterSet *multiclusterv1alpha1.ClusterSet) error {
+func (r *MemberClusterSetReconciler) validateLocalClusterClaim(clusterSet *multiclusterv1alpha1.ClusterSet) error {
```

Comment:
looks like there are duplicate validateLocalClusterClaim method, Could you double check how to reduce those repeated codes? thanks.

Reply:
Okay done.